### ISSUE TITLE
Remove deps from useDebouncedCallback

### ISSUE THREAD

**Issue by anilanar** [xnimorz/use-debounce#21]
The API for `useDebouncedCallback` should be changed because a static analyzer (e.g. eslint) cannot compare `deps` with the callback.

For reference: https://twitter.com/mostruash/status/1109134981182488576

In the end, it should look like:

```
const [debouncedCallback] = useDebouncedCallback(
    useCallback((value) => {
      setValue(value);
    }, [setValue]),
    1000,
); 
```

**Comment by anilanar** [comment]
For reference, I currently use `useDebouncedCallback` as follows:

```
  const myCallback = useDebouncedCallback(
    useCallback(() => {
      /* do some stuff */
    }, [value]),
    500,
  );
```

**Comment by xnimorz** [comment]
Hello, @anilanar 
I think I'll change `useDebouncedCallback` within 1-2 days. 
I'll write here when I publish the release

Thank you for the issue.

**Comment by xnimorz** [comment]
Hi, @anilanar 
I've just published the `2.0.0` release, where the `deps` argument removed. The link to changes https://github.com/xnimorz/use-debounce/blob/master/CHANGELOG.md#200 
Also, I reduced the size of the bundle.

**Comment by anilanar** [comment]
I'll update it and let close this issue once done, you are awesome!
